### PR TITLE
Team Page updates closes #24

### LINKED ISSUE #24: Team page update
There are more team members that are waiting to be added in the dedicated page: https://www.schrodinger-hat.it/team

This is the list of the members: https://github.com/Schrodinger-Hat/Contributors

### PR DESCRIPTION
**Pull request by sparsh-007**
closes #24

### CODE REVIEW

File: src/i18n/messages.json [hunk 1]
```diff
@@ -74,6 +74,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
```
Comment (nicpuppa) [inline src/i18n/messages.json:78]: There is a typo in the name, the correct one is Angy.

File: src/i18n/messages.json [hunk 2]
```diff
@@ -74,6 +74,90 @@
...
+        "name": "Angela Busato",
+        "github": "AngyDev",
+        "github_url": "https://github.com/AngyDev",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
+        "twitter_url": "https://twitter.com/angela_busato",
+        "image": "/img/_dev/Agny.jpg",
```
Comment (nicpuppa) [inline src/i18n/messages.json:84]: Also in the name of the img there is the same typo

File: src/i18n/messages.json [hunk 3]
```diff
@@ -74,6 +74,90 @@
...
+        "secondary_image": "/img/_dev/Agny.jpg",
+        "description": "Sono un'esploratrice, mi piace la ricerca e la sperimentazione negli ambiti più disparati, programmazione, fotografia, stampa 3D, e fai da te sono i primi della mia lista. Nel tempo libero mi piace viaggiare e conoscere nuovi posti, persone, culture ma soprattutto conoscere il cibo, ci sono sempre cose nuove da scoprire.",
+        "website": "",
+        "permalink": "AgnyDev"
```
Comment (nicpuppa) [inline src/i18n/messages.json:88]: Please correct the typo, the correct one is AngyDev

File: src/i18n/messages.json [hunk 4]
```diff
@@ -74,6 +74,90 @@
...
+      },
+      "DanerSound": {
+        "name": "Andre Cristhian",
+        "github": "DannerSound",
+        "github_url": "https://github.com/DannerSound",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
+        "twitter_url": " ",
+        "image": "/img/_dev/Danersound.jpg",
+        "secondary_image": "/img/_dev/Danersound.jpg",
+        "description": "Sono un'esploratrice, mi piace la ricerca e la sperimentazione negli ambiti più disparati, programmazione, fotografia, stampa 3D, e fai da te sono i primi della mia lista. Nel tempo libero mi piace viaggiare e conoscere nuovi posti, persone, culture ma soprattutto conoscere il cibo, ci sono sempre cose nuove da scoprire.",
```
Comment (nicpuppa) [inline src/i18n/messages.json:98]: This is not the right description of DanerSound

File: src/i18n/messages.json [hunk 5]
```diff
@@ -74,6 +74,90 @@
...
+        "secondary_image": "/img/_dev/Agny.jpg",
+        "description": "Sono un'esploratrice, mi piace la ricerca e la sperimentazione negli ambiti più disparati, programmazione, fotografia, stampa 3D, e fai da te sono i primi della mia lista. Nel tempo libero mi piace viaggiare e conoscere nuovi posti, persone, culture ma soprattutto conoscere il cibo, ci sono sempre cose nuove da scoprire.",
+        "website": "",
+        "permalink": "AgnyDev"
+      },
+      "DanerSound": {
+        "name": "Andre Cristhian",
+        "github": "DannerSound",
+        "github_url": "https://github.com/DannerSound",
+        "linkedin_url": "https://www.linkedin.com/in/angela-busato-91145ab1/",
```
Comment (nicpuppa) [inline src/i18n/messages.json:94]: This is not the right LinkedIn url of DanerSound, please use the correct one

File: src/i18n/messages.json [hunk 6]
```diff
@@ -74,6 +74,90 @@
...
+        "permalink": "DanerSound"
+      },
+      "davideimola": {
+        "name": "Davide Imola",
+        "github": "davideimola",
+        "github_url": "https://github.com/davideimola",
+        "linkedin_url": "https://www.linkedin.com/in/davideimola/",
+        "twitter_url": " ",
+        "image": "/img/_dev/davideimola.jpg",
+        "secondary_image": "/img/_dev/Agny.jpg",
```
Comment (nicpuppa) [inline src/i18n/messages.json:109]: Please change this secondary_image url, you should use the same url of image field

File: src/i18n/messages.json [hunk 7]
```diff
@@ -74,6 +74,90 @@
...
+        "twitter_url": "https://twitter.com/angela_busato",
+        "image": "/img/_dev/Agny.jpg",
+        "secondary_image": "/img/_dev/Agny.jpg",
+        "description": "Sono un'esploratrice, mi piace la ricerca e la sperimentazione negli ambiti più disparati, programmazione, fotografia, stampa 3D, e fai da te sono i primi della mia lista. Nel tempo libero mi piace viaggiare e conoscere nuovi posti, persone, culture ma soprattutto conoscere il cibo, ci sono sempre cose nuove da scoprire.",
+        "website": "",
+        "permalink": "AgnyDev"
+      },
+      "DanerSound": {
+        "name": "Andre Cristhian",
+        "github": "DannerSound",
```
Comment (nicpuppa) [inline src/i18n/messages.json:92]: This is not the correct GitHub name, the right one is DanerSound

File: src/i18n/messages.json [hunk 8]
```diff
@@ -74,6 +74,90 @@
...
+        "github_url": "https://github.com/DannerSound",
```
Comment (nicpuppa) [inline src/i18n/messages.json:93]: Please correct also the github_url

File: src/i18n/messages.json [hunk 9]
```diff
@@ -184,6 +268,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
```
Comment (nicpuppa) [inline src/i18n/messages.json:272]: Please check also this second part, there are the same errors as in the first part

File: src/i18n/messages.json (hunk 1 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 2 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 3 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 4 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 5 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 6 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 7 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

File: src/i18n/messages.json (hunk 8 above)
Comment (sparsh-007) [reply src/i18n/messages.json]: done

**Comment by nicpuppa** [comment]
Please add the keyword closes #issue_number on description

**Comment by sparsh-007** [comment]
> Please add the keyword closes #issue_number on description

done

**Comment by nicpuppa** [comment]
Great work @sparsh-007 , thank you!

**Review by nicpuppa — APPROVED** [review]

**Comment by sparsh-007** [comment]
> Great work @sparsh-007 , thank you!

sorry by mistake clicked on review again

**Review by nicpuppa — APPROVED** [review]

**Review by Wabri — APPROVED** [review]

**Pull request merged**